client/components/domains/transfer-domain-step/transfer-domain-precheck.jsx

```diff
@@ -77,6 +78,11 @@ class TransferDomainPrecheck extends React.Component {
 		this.setState( { currentStep: this.state.currentStep + 1 } );
 	};
 
+	refreshStatus = () => {
+		this.setState( { unlockCheckClicked: true } );
+		this.props.refreshStatus();
```

Comment:
Might be an overkill, but I'd rather see the button change once the fetching is finished. This way, feels weird. I believe the `refreshStatus()` passed has an argument `callback`, so you can create a function in `precheck` that just does `this.setState( { unlockCheckClicked: true } );` and pass it in there.